webapp/components/DeleteData/DeleteData.vue

```diff
@@ -62,7 +62,13 @@
               />
             </ds-flex-item>
             <ds-flex-item :width="{ base: '100%', sm: '100%', md: '100%', lg: 1 }">
-              <ds-button icon="trash" danger :disabled="!deleteEnabled" @click="handleSubmit">
+              <ds-button
+                icon="trash"
+                danger
+                filled
+                :disabled="!deleteEnabled"
+                @click="handleSubmit"
+              >
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:19:20Z" title="Tuesday, January 14th 2020, 6:19:20 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -62,7 +62,13 @@
               />
             </ds-flex-item>
             <ds-flex-item :width="{ base: '100%', sm: '100%', md: '100%', lg: 1 }">
-              <ds-button icon="trash" danger :disabled="!deleteEnabled" @click="handleSubmit">
+              <ds-button
+                icon="trash"
+                danger
+                filled
+                :disabled="!deleteEnabled"
+                @click="handleSubmit"
+              >
```


why is this still `ds-button`?
actually, there are 10 files for me locally that still include `ds-button`... I will have a look at the other `PR` you put in... maybe there were bigger changes needed?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:43:50Z" title="Wednesday, January 15th 2020, 8:43:50 am +01:00">Jan 15, 2020</time>_

---


yes, that's in the second PR. I know it's a bit confusing, I'll try to find a better solution next time than splitting things up into two PRs... sorry! 🤷‍♀